### DISCUSSION TITLE
Create a developer Tips section

### DISCUSSION THREAD

**Discussion by webvs2** [withastro/docs#8576] [Ideas]
在实际的开发中，程序的编写难度一般比文档上的大一些，且跟相关的第三方插件结合使用，其中的一些开发注意点会被忽略，希望有这一个栏目，能吧每个人忽略到的地方说出来，来更好的帮助其他开发者。

In the actual development, the difficulty of writing the program is generally larger than the document, and it is used in combination with related third-party plug-ins, some of the development points will be ignored, I hope there is a column, can everyone ignore the place to say it, to better help other developers.

例如。我遇到了i18next的开发中的一个小注意点，就是Imges的路径文件，写默认的 `images/xxx/xx.png` 就可以展示，当切换的时候这个图片就会丢失，应该写成 `/images/xxx/xx.png` 相对路径。

For example. I encountered a small note in the development of i18next, that is, the Imges path file, write the default 'images/xxx/xx.png' can be displayed, when the switch this image will be lost, should be written '/images/xxx/xx.png' relative path.

**Comment by at-the-vr** [comment]
https://astro-tips.dev/ is one of the sites to record plethora of developer tips. You can contribute your tips in there as well 😉

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

And, as Atharva said, we expect these kind of tips to go to Astro Tips instead of our docs!

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
